src/main/java/libraryapp/config/AppConfig.java

```diff
@@ -51,7 +53,7 @@ public LocalSessionFactoryBean getSessionFactory() {
         props.put("hibernate.dialect", env.getProperty("hibernate.dialect"));
 
         factoryBean.setHibernateProperties(props);
-        factoryBean.setAnnotatedClasses(User.class, Book.class);
+        factoryBean.setAnnotatedClasses(User.class, Book.class, Author.class, Rent.class);
```

Comment:
we can use this line here:
factoryBean.setPackagesToScan("libraryapp.entity");

Reply:
thanks :)